test/fail/ok/try-finally.tc.ok

```diff
@@ -1,5 +1,6 @@
-try-finally.mo:10.17-10.31: type error [M0200], a cleanup clause must not send messages
-try-finally.mo:16.17-16.39: type error [M0200], a cleanup clause must not send messages
+try-finally.mo:10.26-10.29: type error [M0047], send capability required, but not available
```

Comment:
One could add an `.md` explanation file to give users a hint how to fix this (`M0047`).

Reply:
I'd be happy to do that if you think that's a good idea.

Reply:
I've merged into your branch. Up to you if you want to add an .md file... If you do, there's some sections in the doc about where sends are allowed that you could re-use... That could also be a completely separate PR of course.